docker/Dockerfile.release

```diff
@@ -1,4 +1,11 @@
 FROM ubuntu:xenial
+RUN apt-get update && \
```

Comment:
copyright

Reply:
curious about that too, I think it's covered at the repo level anyway - and for short files it's a lot of overhead ?